### PR TITLE
Add configuration for global search(RET-2462)

### PR DESCRIPTION
**Pull request by dharmendrak**
### JIRA link (if applicable) ###

RET-2462

### Change description ###

Add configuration for global search

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ ] No
```

### CODE REVIEW

File: definitions/json/SearchParty.json [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+    "SearchPartyCollectionFieldName": "respondentCollection",
+    "SearchPartyName": "respondent_name",
+    "SearchPartyEmailAddress": "respondent_email",
+    "SearchPartyAddressLine1": "respondent_address.AddressLine1",
+    "SearchPartyPostCode": "respondent_address.PostCode"
+  },
+  {
+    "LiveFrom": "01/01/2020",
+    "CaseTypeID": "ET_EnglandWales",
+    "SearchPartyCollectionFieldName": "repCollection",
```
Comment (fergusoshea) [inline definitions/json/SearchParty.json:24]: This is missing on ET_Scotland

File: definitions/json/AuthorisationCaseField.json [hunk 2]
```diff
@@ -24286,5 +24310,329 @@
...
     "UserRole": "citizen",
     "CRUD": "CRU"
+  },
+  {
+    "LiveFrom": "01/01/2017",
+    "CaseTypeId": "ET_EnglandWales",
+    "CaseFieldID": "SearchCriteria",
+    "UserRoles": [
+      "GS_profile",
+      "caseworker-employment-legalrep-solicitor",
```
Comment (fergusoshea) [inline definitions/json/AuthorisationCaseField.json:24320]: Other than "GS_profile", none of these other UserRoles are used in ET_Scotland

**Review by fergusoshea — APPROVED** [review]

File: definitions/json/AuthorisationCaseField.json [hunk 3]
```diff
@@ -21767,6 +21767,18 @@
     "UserRole": "caseworker-employment-api",
     "CRUD": "CRUD"
   },
+  {
+    "CaseTypeId": "ET_EnglandWales_Listings",
```
Comment (hsjhita) [inline definitions/json/AuthorisationCaseField.json:21771]: confused as to why these fields are being added to the Listings and Multiple caseTypes

File: definitions/json/CaseField.json [hunk 4]
```diff
@@ -5408,5 +5422,115 @@
...
+    "Label": "Claimant",
+    "FieldType": "Text",
+    "SecurityClassification": "Public"
+  },
+  {
+    "CaseTypeID": "ET_EnglandWales_Multiple",
+    "ID": "respondent",
+    "Label": "Respondent",
+    "FieldType": "Text",
+    "SecurityClassification": "Public"
```
Comment (hsjhita) [inline definitions/json/CaseField.json:5534]: reason why listings and multiples are having these fields added even though theres no code to populate?

File: definitions/json/CaseEvent.json [hunk 5]
```diff
@@ -1355,5 +1357,44 @@
...
+    "PreConditionState(s)": "*",
+    "PostConditionState": "*",
+    "CallBackURLAboutToSubmitEvent": "${ET_COS_URL}/global-search-migration/about-to-submit",
+    "CallBackURLSubmittedEvent": "${ET_COS_URL}/global-search-migration/submitted",
+    "SecurityClassification": "Public",
+    "ShowEventNotes": "N",
+    "ShowSummary": "N"
+  },
+  {
+    "CaseTypeID": "ET_EnglandWales_Multiple",
```
Comment (hsjhita) [inline definitions/json/CaseEvent.json:1375]: again, do we need this event on multiples and listings

**Review by JackReevies — APPROVED** [review]

**Review by hsjhita — APPROVED** [review]

**Pull request merged**